modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -370,6 +442,40 @@ private NotificationTemplate _addNotificationTemplate(
 				Collections.singletonList(objectField.getObjectFieldId())));
 	}
 
+	private NotificationTemplate _addNotificationTemplateScopeSite(
+			String body, String editorType, boolean singleRecipient,
+			Map<Locale, String> to)
+		throws Exception {
+
+		ObjectField objectField = objectFieldLocalService.getObjectField(
+			scopeSiteObjectDefinition.getObjectDefinitionId(),
+			"attachmentObjectField");
```

Comment:
Hey @rijema use a `fetchObjectField` so we can remove the attachmentObjectField of `scopeSiteObjectDefinition`, so when the objectField is null we pass an empty list at [this line](https://github.com/liferay-objects/liferay-portal/pull/3143/files#diff-eca215044a4eb53b9507b221f69a273025af4136f243a39d4b6c01cdccb5dcf8R476)